cookbook/case_studies/ml_training/house_price_prediction/house_price_predictor.py

```diff
@@ -18,13 +18,15 @@
 # First, import all the required libraries.
 import typing
 
+import os
+import flytekit
```

Comment:
why is this line required?

Reply:
```
    working_dir = flytekit.current_context().working_directory
    fname = os.path.join(working_dir, f"model-{loc}.joblib.dat")
```
This isn't required, but then it makes more sense to me.